14-draft.json

```diff
@@ -975,6 +975,47 @@
               "value"
             ]
           }
+        },
+        "network_traffic": {
+          "description": "The current network bandwidth usage",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "value": {
+                "description": "The measurement value",
+                "type": "number"
+              },
+              "unit": {
+                "description": "The measurement unit",
+                "type": "string",
+                "enum": [
+                  "packets/s",
+                  "bit/s",
+                  "kbit/s",
+                  "Mbit/s",
+                  "Gbit/s",
+                  "Byte/s",
+                  "kByte/s",
+                  "MByte/s",
+                  "GByte/s"
```

Comment:
I would propose to leave away the units and always specify all values either as bits per second or as bytes per second (I tend to favor bits per second, it's more common in networking). It makes parsing easier and causes less ambiguity (especially the MB vs MiB ambiguity). Otherwise a parser will get complex quickly.

Do we need packets per second?

Reply:
The packet rate is also interesting, it may be limited by the capability of the router and together with the data rate it shows the average packet size. But then it must be possible to specify both values.

Reply:
@dbrgn I agree, having only bits per second seems like the better choice. Though I think having a packet rate makes sense as well, but e.g. as a separate field, like @koalatux suggests.